src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -39,16 +38,21 @@ public String addFacilityPage(Model model) {
         facility.setFacilityId(110L);
         facility.setMaterialId(50L);
         model.addAttribute(facility);
-        return "AddFacility";
+        return "addFacility";
     }
 
-    @RequestMapping("/saveFacility")
-    public String saveFacility(Facility facility) {
+    @PostMapping("/saveFacility")
+    public String saveFacility(Facility facility, Model model) {
         facilityService.save(facility);
+        RecyclableMaterial recyclableMaterial = new RecyclableMaterial();
+        Facility facilityModel = new Facility();
+        recyclableMaterial.setMaterialName("");
+        facilityModel.setFacilityAddress("");
+        model.addAttribute(recyclableMaterial);
         return "index";
     }
 
-    @GetMapping("/facility/")
```

Comment:
Since you are using request parameters to pass information to the controller, you do not need the extra slash at the end. I would recommend changing this since it caused some Runtime issues.